MARKETPLACE.md

```
@@ -0,0 +1,203 @@
+
+```bash
+# From specific branch
+/plugin marketplace add jmagar/unraid-mcp#main
+
+# From specific commit
+/plugin marketplace add jmagar/unraid-mcp#abc123
+```
+
+## Plugin Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)